rpc/common/moc_common_nodeinfo.proto

```diff
@@ -44,13 +54,12 @@ message NodeInfo {
 	OperatingSystemType ostype = 6;
 	Status status = 7;
 	int64 uptime = 8;
+	OperatingSystem osInfo = 9;
 }
 
-message Resources {   
+message Resources {
 	Processor processor = 1;
 	PhysicalMemory memory = 2;
-	GPU gpu = 3;
-	OperatingSystem osInfo = 4;
```

Comment:
Was this unexpectedly removed? Would this change cause a backwards compatibility issue?

Reply:
@JocelynBerrendonner osInfo is not consumed by anyone at the moment as confirmed by Madhan. So we decided to move it outside of Resources and add it in NodeInfo.

Reply:
vmCapabilities should retain its number, right?

Reply:
@in3xes I'm not so sure about it. Since we're removing one field, I thought using 4 for vmCapabilities should be fine.

Reply:
https://github.com/microsoft/moc/blob/f5b55407a9b230a9cac53169911c679567ffb6df/rpc/cloudagent/network/virtualnetwork/moc_cloudagent_virtualnetwork.proto#L40

I think this is the right way to mark it reserved

Reply:
Done.